app/assets/javascripts/jquery.wayfinding.js

```diff
@@ -19,6 +19,7 @@
 (function ($) {
 	'use strict';
 
+	var animating = false;
```

Comment:
This should be in the 'defaults' object as that seems to be where we're storing everything.